src/components/app/MenuButtons/MetaInfo.css

```diff
@@ -15,3 +15,59 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
+.metaInfoPanel {
+  width: 350px;
+  font-size: 12px;
+  line-height: 20px;
+}
+
+.metaInfoPanel,
+.metaInfoPanel .arrowPanelArrow::before {
+  background-color: white;
+}
+
+.metaInfoSection {
+  margin-top: 7px;
+  margin-bottom: 12px;
+}
+
+.metaInfoLabel,
+.metaInfoWideLabel {
+  display: inline-block;
+  width: 120px;
+  color: var(--grey-50);
+}
+
+.metaInfoWideLabel {
+  width: 140px;
+}
+
+.visualMetricsLabel {
+  display: inline-block;
+  width: 145px;
+  color: var(--grey-50);
+}
+
+.metaInfoList {
+  padding: 0;
+
+  /* Explanations about this margin line:
+   * - we need to reset it anyway, because we need to remove the default style
+   *   for <ul> tags.
+   * - -19px moves the list up, next to the label.
+   * - 122px is 120px (the width of the label) + a few pixels so that this looks
+   *   better.
+   */
+  margin: -19px 0 0 122px;
```

Comment:
This is a change and (I think) a simplification from what we had before (a mix of padding / margin and px / rem).